### PR TITLE
feat, style: improve FilePicker UI, Improve preferences & some loading dialogs.

### PR DESCRIPTION
**Pull request by wolfdeveloper01**
FilePicker:
Screenshot_20240811-165347
Screenshot_20240811-165406

dialogs:

IMG-20240802-WA0030
IMG-20240802-WA0032
-util 

I will not show the preferences, because there are no significant visual changes.

### CODE REVIEW

**Comment by PranavPurwar** [comment]
Can you please show if there are any benefits in using your library instead of the original one? Screenshots describing the changes are also welcome.

File: app/src/main/java/mod/jbk/util/OldResourceIdMapper.java [hunk 1]
```diff
@@ -2197,7 +2197,7 @@ public static int getDrawableFromOldResourceId(int oldId) {
             case 2131166281:
                 return R.drawable.yellow_arrow_down;
             case 2131166282:
-                return R.drawable.bottom_shadow;
+                return R.drawable.img_nocontent;
```
Comment (remaker17) [inline app/src/main/java/mod/jbk/util/OldResourceIdMapper.java:2200]: Why did you change it?

File: app/src/main/res/layout/fragment_settings_appearance.xml [hunk 2]
```diff
@@ -52,40 +52,40 @@
                 <TextView
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:textColor="?attr/colorPrimary"
+                    android:textColor="?colorPrimary"
```
Comment (remaker17) [inline app/src/main/res/layout/fragment_settings_appearance.xml:55]: Please do not make useless changes.

**Review by remaker17 — CHANGES_REQUESTED** [review]

**Comment by PranavPurwar** [comment]
> > Can you please show if there are any benefits in using your library instead of the original one? Screenshots describing the changes are also welcome.
> 
> Sobre usar a lib dos components, bom, o código fica menor, e fica mais padronizado
> 
> IMG-20240802-WA0030 IMG-20240802-WA0031 IMG-20240802-WA0032 -util

In my opinion, the file picker dialog is still not completely material you themed, so it'd be nice if you can make it consistent with the rest of the app.

File: app/src/main/java/mod/jbk/util/OldResourceIdMapper.java (hunk 1 above)
Comment (NickolasBressy) [reply app/src/main/java/mod/jbk/util/OldResourceIdMapper.java:2200]: o arquivo bottom_change fazia parte da lib antiga do FilePicker, como foi tirada ,não tem mais o arquivo

**Comment by NickolasBressy** [comment]
> > > Você pode mostrar se há algum benefício em usar sua biblioteca em vez da original? Capturas de tela descrevendo as mudanças também são bem-vindas.
> > 
> > 
> > Sobre usar a lib dos components, bom, o código fica menor, e fica mais padronizado
> > IMG-20240802-WA0030 IMG-20240802-WA0031 IMG-20240802-WA0032-util
> 
> Na minha opinião, a caixa de diálogo do seletor de arquivos ainda não está completamente de acordo com o tema que você definiu, então seria legal se você pudesse torná-la consistente com o restante do aplicativo.

bom, ela não é minha, mas posso falar com o cara da lib

File: app/src/main/res/layout/fragment_settings_appearance.xml (hunk 2 above)
Comment (NickolasBressy) [reply app/src/main/res/layout/fragment_settings_appearance.xml:55]: ok, mas e agora, o que faço

File: app/src/main/java/mod/jbk/util/OldResourceIdMapper.java (hunk 1 above)
Comment (NickolasBressy) [reply app/src/main/java/mod/jbk/util/OldResourceIdMapper.java:2200]: > o arquivo bottom_change fazia parte da lib antiga do FilePicker, e já que eu tirei a lib, o arquivo não é mais encontrado

File: app/src/main/res/layout/fragment_settings_appearance.xml (hunk 2 above)
Comment (remaker17) [reply app/src/main/res/layout/fragment_settings_appearance.xml:55]: Nothing

File: app/src/main/java/mod/trindadedev/tools/apk/ApkUtils.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Formatter;
+
+public class ApkUtils {
+
+    private String apkPath;
+
+    public ApkUtils() {
+    }
```
Comment (remaker17) [inline app/src/main/java/mod/trindadedev/tools/apk/ApkUtils.java:14]: Unnecessary code.

**Comment by HanzoDev1375** [comment]
افقانی ها جمع شدن

**Comment by aquilesTrindade13** [comment]
🤔

**Review by remaker17 — APPROVED** [review]

**Comment by remaker17** [comment]
Can you reduce the size of text in file picker dialog?

**Comment by trindadeDev13** [comment]
> Can you reduce the size of text in file picker dialog? 

Of course.

**Comment by trindadeDev13** [comment]
@remaker17, what sizes do you think would look best?

**Comment by remaker17** [comment]
> @remaker17, what sizes do you think would look best?

Just decrease by 2.
e.g 18 -> 16

**Comment by ilyassesalama** [comment]
I think your device is showing fonts a bit bigger than they should. I tested on 2 devices and the font size looks great.

> Can you reduce the size of text in file picker dialog?

**Comment by trindadeDev13** [comment]
> > @remaker17, what sizes do you think would look best?
> 
> Just decrease by 2. e.g 18 -> 16

okay, I'll do some tests.

**Comment by remaker17** [comment]
> I think your device is showing fonts a bit bigger than they should. I tested on 2 devices and the font size looks great.
> 
> > Can you reduce the size of text in file picker dialog? 
> 
> 

It's not mine screenshot. Also why then on other screenshots font size looks okay?

**Comment by remaker17** [comment]
why did you merge?
why didn't you answer my question?
